web/apps/labelstudio/src/config/ApiConfig.js

```diff
@@ -53,8 +53,9 @@ export const API_CONFIG = {
     updateMLBackend: "PATCH:/ml/:pk",
     deleteMLBackend: "DELETE:/ml/:pk",
     trainMLBackend: "POST:/ml/:pk/train",
-    predictWithML: "POST:/ml/:pk/predict",
+    predictWithML: "POST:/ml/:pk/predict",    
```

Comment:
```suggestion
    predictWithML: "POST:/ml/:pk/predict",
```

The linters should be taking care of this very soon!